Problem is this PR is blowing up the tests:
```
Traceback (most recent call last):
  File "code/setup.py", line 124, in <module>
    download_url="https://github.com/dmwm/WMCore/tarball/%s" % wmcore_version
  File "/build/cmsbld/jenkins-workarea/workspace/DMWM-PR-unittests/SLICE/0/label/slc6_amd64-cloud-tiny/deploy/1.1.1.pre3/sw/slc6_amd64_gcc493/external/python/2.7.11-comp/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/build/cmsbld/jenkins-workarea/workspace/DMWM-PR-unittests/SLICE/0/label/slc6_amd64-cloud-tiny/deploy/1.1.1.pre3/sw/slc6_amd64_gcc493/external/python/2.7.11-comp/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/build/cmsbld/jenkins-workarea/workspace/DMWM-PR-unittests/SLICE/0/label/slc6_amd64-cloud-tiny/deploy/1.1.1.pre3/sw/slc6_amd64_gcc493/external/python/2.7.11-comp/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/build/cmsbld/jenkins-workarea/workspace/DMWM-PR-unittests/SLICE/0/label/slc6_amd64-cloud-tiny/code/setup_test.py", line 275, in run
    import WMQuality.TestInit
  File "/build/cmsbld/jenkins-workarea/workspace/DMWM-PR-unittests/SLICE/0/label/slc6_amd64-cloud-tiny/code/src/python/WMQuality/TestInit.py", line 25, in <module>
    from WMCore.Agent.Configuration import Configuration
ImportError: No module named Agent.Configuration
```

--- a/.gitignore
+++ b/.gitignore
@@ -20,3 +20,5 @@ setup.sh
 setup_environment_only.sh
 shutdown.sh
 tester_runner_bash.sh
+MANIFEST
+dist/

--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,0 +1,1 @@
+include setup_*.py

--- a/setup.py
+++ b/setup.py
@@ -4,6 +4,7 @@ Build, clean and test the WMCore package.
 """
 from __future__ import print_function
 
+import imp
 import os
 import os.path
 from distutils.core import Command, setup
@@ -95,8 +96,27 @@ DEFAULT_PACKAGES = list_packages(['src/python/Utils',
                                   'src/python/WMQuality',
                                   'src/python/PSetTweaks'])
 
+# Divine out the version of WMCore from WMCore.__init__, which is bumped by
+# "bin/buildrelease.sh"
+
+# Obnoxiously, there's a dependency cycle when building packages. We'd like
+# to simply get the current WMCore version by using
+# from WMCore import __version__
+# But PYTHONPATH isn't set until after the package is built, so we can't
+# depend on the python module resolution behavior to load the version.
+# Instead, we use the imp module to load the source file directly by
+# filename.
+wmcore_root = get_path_to_wmcore_root()
+wmcore_package = imp.load_source('temp_module', os.path.join(wmcore_root,
+                                                            'src',
+                                                            'python',
+                                                            'WMCore',
+                                                            '__init__.py'))
+wmcore_version = wmcore_package.__version__
+
 setup(name='wmcore',
-      version='1.0',
+      version=wmcore_version,
+      maintainer='CMS DMWM Group',
       maintainer_email='hn-cms-wmDevelopment@cern.ch',
       cmdclass={'deep_clean': CleanCommand,
                 'coverage': CoverageCommand,
@@ -107,4 +127,7 @@ setup(name='wmcore',
       # base directory for all our packages
       package_dir={'': 'src/python/'},  # % get_path_to_wmcore_root()},
       packages=DEFAULT_PACKAGES,
-      data_files=list_static_files())
+      data_files=list_static_files(),
+      url="https://github.com/dmwm/WMCore",
+      download_url="https://github.com/dmwm/WMCore/tarball/%s" % wmcore_version
+      )

